docs/components/utils/examples/embeddings.ipynb

```diff
@@ -2,409 +2,103 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "7ef4d402-6662-4a26-b612-35b542066487",
-   "metadata": {
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   },
+   "id": "249b4058",
```

Comment:
High level comment about this file -- would be good to have a general how-to guide about create a vector store with document embeddings, and then point to the API ref for examples on how to load specific providers.

Also, would be useful if there are docs associated with the codeblocks.

Reply:
there is the vectorstores notebook, is that what you meant?